### PR TITLE
Add more unit tests for AppSpider report parser

### PR DESCRIPTION
**Pull request by damiencarol**
## Work
- [X] add unit test file with one vulnerability
- [X] add unit test for CWE
- [X] add unit test for null

### CODE REVIEW

File: dojo/tools/appspider/parser.py [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
     def __init__(self, filename, test):
+        self.items = []
+
+        if filename is None:
+            return
 
         if "VulnerabilitiesSummary.xml" not in str(filename):
-            raise NamespaceErr('Please ensure that you are uploading AppSpider\'s VulnerabilitiesSummary.xml file.'
-                               'At this time it is the only file that is consumable by DefectDojo.')
+            logging.warning('Please ensure that you are uploading AppSpider\'s VulnerabilitiesSummary.xml file.'
```
Comment (valentijnscholten) [inline dojo/tools/appspider/parser.py:17]: are you sure this is intended behaviour? the warning will not be seen by the end user and the import will continue anyway instead of raising an exception.

Comment (damiencarol) [reply dojo/tools/appspider/parser.py:17]: this check is useless. it force the code to have a real file with a fixed name. it is not what we want.
we want to load data evene without a real file on the disk

Comment (valentijnscholten) [reply dojo/tools/appspider/parser.py:17]: ah, now I see. I thought it was checking for some XML node to be present. But checking the filename makes no sense, so the whole ìf` block can be removed I think.

Comment (damiencarol) [reply dojo/tools/appspider/parser.py]: Don't ask me twice to remove useless code 😄 !
Made the change requested.

**Review by valentijnscholten — APPROVED** [review]
approved assuming you'll fix the flake8 violation :-)

**Review by madchap — APPROVED** [review]
